cli/pcluster/config/param_types.py

```diff
@@ -34,10 +33,11 @@
 def handle_no_section_error(func):
     @functools.wraps(func)
     def wrapper(*args, **kwargs):
+        param = args[0]  # "self" argument
```

Comment:
I'm afraid this is not safe cause the first arg is not always self but it depends how you invoke this function.
Why don't we move this functions outside of the PclusterConfig class?

Reply:
The wrapper has been totally removed now; the exception is handled directly from PClusterConfig